src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,26 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## March 9, 2023
+
+### AEP React Native libraries 2.0.0
+
+* Updated all the AEP React Native libraries to use MobileCore 2.x for Android and compatible extensions.
```

Comment:
"Updated all the React Native libraries to use Mobile Core 2.x for Android and compatible extensions."

(Mobile Core _is_ two words, right?)

Reply:
MobileCore is the extension name, so it should be one word.

Reply:
I think in sentences we can refer to it as Mobile Core since it refers to the Core SDK , except when we refer to the "MobileCore" class.

Reply:
https://github.com/AdobeDocs/aep-mobile-sdkdocs/pull/262/files#diff-f069d95da62525e6f5ca4630ece2eb4b391e3a42f736f381245ae50546975706R27
We use `MobileCore` in Flutter release notes, so I just port the same to the RN release notes.`MobileCore` is the Android Core's public API class, so I think it represents the Android Core extension as well. Let me know if you have other options, I can update both places in a new PR.